Apply-For-Me-UI/src/pages/contact_us/ContactUs.jsx

```diff
@@ -1,7 +1,48 @@
 import React from "react";
+import ContactUsForm from "./components/ContactUsForm";
+import LeftDetails from "./components/LeftDetails";
+import "./components/ContactStyle.css";
+import Nav from "../../components/nav/Nav";
+import Footer from "../../components/footer/Footer";
+
 
 const ContactUs = () => {
-    return <div>Conatct us</div>;
+    return (
+        <div className="contact">
+          <div className="contact-nav-wrapper">
+            <Nav/>
+          </div>
```

Comment:
- PLease remove the <div className="conttact-nav-wrapper> container. the nav component has its own width.